src/Components/Milestones/MilestonesTimeline.jsx

```diff
@@ -0,0 +1,238 @@
+    time: 'June,2021',
+  },
+  {
+    title: 'SDV 2.0',
+    body: 'IEEE COMPUTER SOCIETY SYP MGAB in collaboration with IEEE SB NIT Durgapur and IEEE SB Jadavpur University hosted the IEEE TechX Congress 2020: Eastern India Techno-Management Leadership Summit on 28th Feb - 1st March 2020, where it assimilated all students and Young IntelliSys:Machine Learning Symposium was a huge success with a colossal turnout of over 150 enthusiasts who were too eager to learn and explore.IEEE SB NIT Durgapur extends its heartfelt gratitude to our guest speakers, Dr. Fakhral Yusoff and Mr. Shivam Abhilash for leaving us all dazzled and especially to all the participants for their overwhelming response.Professionals from various IEEE Sections and students branches.',
+    time: 'May,2021',
+  },
+  {
+    title: 'IntelliSys',
+    body: 'IEEE COMPUTER SOCIETY SYP MGAB in collaboration with IEEE SB NIT Durgapur and IEEE SB Jadavpur University hosted the IEEE TechX Congress 2020: Eastern India Techno-Management Leadership Summit on 28th Feb - 1st March 2020, where it assimilated all students and Young Professionals from various IEEE IntelliSys:Machine Learning Symposium was a huge success with a colossal turnout of over 150 enthusiasts who were too eager to learn and explore.IEEE SB NIT Durgapur extends its heartfelt gratitude to our guest speakers, Dr. Fakhral Yusoff and Mr. Shivam Abhilash for leaving us all dazzled and especially to all the participants for their overwhelming response.Sections and students branches. ',
+    time: 'Jan,2021',
+  },
+  {
+    title: 'IntelliSys',
+    body: 'IEEE COMPUTER SOCIETY SYP MGAB in collaboration with IEEE SB NIT Durgapur and IEEE SB Jadavpur University hosted the IEEE TechX Congress 2020: Eastern India Techno-Management Leadership Summit on 28th Feb - 1st March 2020, where it assimilated all students and Young IntelliSys:Machine Learning Symposium was a huge success with a colossal turnout of over 150 enthusiasts who were too eager to learn and explore.IEEE SB NIT Durgapur extends its heartfelt gratitude to our guest speakers, Dr. Fakhral Yusoff and Mr. Shivam Abhilash for leaving us all dazzled and especially to all the participants for their overwhelming response. Professionals from various IEEE Sections and students branches. ',
+    time: 'April,2020',
+  },
+  {
+    title: 'IntelliSys',
+    body: 'IEEE COMPUTER SOCIETY SYP MGAB in collaboration with IEEE SB NIT Durgapur and IEEE SB Jadavpur University hosted the IEEE TechX Congress 2020: Eastern India Techno-Management Leadership Summit on 28th Feb - 1st March 2020, where it assimilated all students and Young Professionals from various IEEE Sections and students branches.',
+    time: 'Aug,2020',
+  },
+];
+
```

Comment:
Make a separate js file inside the Data folder and import these informations from there.

Reply:
✅